Cargo.toml

```diff
@@ -1,7 +1,11 @@
 [workspace]
 resolver = "2"
 members = [
-    "cortex-r", "cortex-r-examples",
+    "cortex-r",
+    "cortex-r-examples",
     "cortex-r-rt",
-    "cortex-r-semihosting"
+    "arm-semihosting"
+]
+exclude = [
+    "arm-targets"
```

Comment:
Can I ask why you excluded this? (Mostly interested since it lead to new CI jobs)

Reply:
because it's a host crate not a cross-compiled crate (it uses libstd).

If you don't exclude it `cargo build --target <cross-target>` will explode with many errors.

It's very annoying, but you just can't reasonably have a multi-target workspace.